packages/react-dom-bindings/src/events/ReactDOMEventReplaying.js

```diff
@@ -537,7 +537,7 @@ export function retryIfBlockedOn(
 
   if (enableFormActions) {
     // Check the document if there are any queued form actions.
-    const root = unblocked.getRootNode();
```

Comment:
Hm... I wonder if it's also an issue if `unblocked` is the `document`.

Reply:
Yea, you have to check if these are already documents and if so just use them as is, otherwise `getRootNode()`.

Reply:
How would you check that? `unblocked === document` doesn't work in the tests, but I think I can assume if `ownerDocument` is undefined, then this is the document?

Reply:
Tried it, but the fizz tests are failing 
<img width="724" alt="Screenshot 2024-02-22 at 11 16 22 AM" src="https://github.com/facebook/react/assets/2440089/26d70a14-399d-4b21-b06a-b12bb78c2ff2">